client/src/main.scss

```diff
@@ -0,0 +1,379 @@
+body {
+  font: {
+   family: Arial, Helvetica, sans-serif;
+   size:15px;
+  }
+  line-height: 1.5;
+  padding: 0;
+  margin: 0;
+  background: #f4f4f4;
+}
+
+.mylist {
+	list-style:none;
+    padding:0;
+	margin:0;
+	//background-color: red !important
+}
+#transform {
+  text-transform: capitalize;
+}
+
+#messages_compose {
+  margin-bottom: 50px;
+}
+
+.viewMessageBoard {
+	overflow-y: scroll;
+	height: 600px;
+}
+
+.sendMessageDiv {
+	margin-top: 4%;
+}
+
+.sendMessageInput {
+	border: 1px solid #2e353d;
+	padding: 1%;
+}
+
+/* Global */
+.container {
+  width: 80%;
+  margin: auto;
+  overflow: hidden;
+}
+
+.google-button{
+	margin-top: 20px;
+	background-color: white !important;
+	color: black !important;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  text-decoration-style: none;
+  text-decoration: none;
+}
+
+/* Header */
+header {
+  background: #35424a;
+  color: #ffffff;
```

Comment:
Since this is `scss`, you should assign `#ffffff` to a variable so it can be re-used throughout the file